website/src/app/kb/architecture/security-controls/readme.mdx

```diff
@@ -0,0 +1,74 @@
+import SupportOptions from "@/components/SupportOptions";
+
+# Architecture: Security Controls
+
+Firezone employs a few different security controls to keep data secure in
+transit and at rest.
+
+## Overview of cryptography used
+
+Below is a table of cryptography used and to which contexts they apply.
+
+| Cryptography                                             | Context         | Notes                                                                                                                                        |
+| -------------------------------------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| AES-256-GCM                                              | Data at rest    | Used to encrypt sensitive data that needs to be persisted, such as authentication tokens.                                                    |
```

Comment:
Should we list hashing functions? We also use `SHA3-256` (hashing of tokens and secrets) and `Argon2` (password hashing, but enabled on dev envs only).

Reply:
done